NightscoutServiceKit/RemoteCommands/RemoteCommandSource.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  RemoteCommandSource.swift
+//  NightscoutServiceKit
+//
+//  Created by Bill Gestrich on 2/25/23.
+//  Copyright © 2023 LoopKit Authors. All rights reserved.
+//
+
+import LoopKit
+
```

Comment:
This will get more methods in future PRs such as methods that fetch pending remote commands that we poll for.